### ISSUE TITLE
Cannot find appropriate database

### ISSUE THREAD

**Issue by ParvaShah** [yahoo/open_nsfw#55]
Hi, I'm working on my final year project which is detecting nsfw content in images and further for videos (if possible). My problem is that i cannot find any dataset for working so if you could please help me out with this problem by giving me access to some amount of your data will really help me in my project .   I really need data of about **(150-200 gb)** to make my algorithm more precise.

**Comment by jienimi** [comment]
随着先进的医学影像设备的不断发展和图像处理算法的不断进步，医学影像技术得到迅猛发展，使诊断和治疗水平都有很大提高。眼底图像处理系统的开发，可以辅助眼科医生更好地进行疾病诊断与治疗。文中通过对视网膜图像处理平台的设计与实现，旨在为科研工作者提供一个算法交流、复用的平台，同时还可以提高视网膜图像处理系统软件开发效率。 文中针对视网膜图像处理平台的设计与实现，对视网膜图像处理算法、视网膜图像处理平台框架的设计与平台的实现进行了研究与开发。在视网膜图像处理算法方面，对视盘的定位、硬性分泌物的检测、视网膜图像亮度和对比度归一化校正、视网膜图像血管提取、视网膜图像拼接等算法进行了研究及实现；在视网膜图像处理软件平台框架的设计与平台实现方面，采用面向对象及设计模式软件技术，对视网膜图像处理平台框架的设计目标、需求分析、应用层框架、功能模块做了详细的分析，实现了视网膜图像处理平台。最后，使用Visual C++软件开发工具、基于ACCESS数据库和ADO的数据库访问技术，开发了一个具有一定病人诊断信息管理功能的视网膜图像处理原型系统。 该视网膜图像处理平台具有一定使用价值，且实现的视网膜图像处理原型系统为眼科医生提供了良好的辅助工具。
【学位授予单位】：西安理工大学
【学位级别】：硕士
【学位授予年份】：2007
【分类号】：TP391.41

**Comment by jienimi** [comment]
【学位授予单位】：西安理工大学
【学位级别】：硕士
【学位授予年份】：2007
【分类号】：TP391.41

**Comment by zippeurfou** [comment]
Hey @ParvaShah did you solve your issue?
I'd be interested as well. so far I found https://sites.google.com/site/pornographydatabase/.

**Comment by ParvaShah** [comment]
No, i was not able to get hold of enough images for my project. I scrapped
webpages like Wikimedia for images of penis and vagina. Did you get access
to download images from pronographydatabase?

**Comment by loretoparisi** [comment]
@zippeurfou thanks for the link. While this dataset covers a lot of cases, it is missing sketches, cartoons and images of graphic violence and/or pornography etc. An approach could be using Flicker as described here http://sergeykarayev.com/files/1311.3715v3.pdf
